packages/react-inbox/src/components/Messages/styled.ts

```diff
@@ -98,7 +98,7 @@ export const Footer = styled.div(({ theme }) =>
 );
 
 export const Line = styled.div(({ theme }) => ({
-  backgroundColor: theme?.brand?.colors?.primary ?? "#9121c2",
+  backgroundColor: theme?.brand?.colors?.primary ?? "#F9FAFB",
```

Comment:
isn't 9121c2 a default we use at a bunch of places in frontend?

Reply:
that's not the default we want for background color afaik

Reply:
good catch. I wasn't intended to change this.